### ISSUE TITLE
Feature Request: Tiled GIFs

### ISSUE THREAD

**Issue by shamesung** [Waitsnake/AnimatedGif#21]
Tiled GIFs

From this:
![screen shot 2017-07-08 at 10 13 48 pm](https://user-images.githubusercontent.com/6269751/27990953-deb20f90-642c-11e7-92cb-793b7cced080.png)

To this:
![screen shot 2017-07-08 at 10 20 43 pm](https://user-images.githubusercontent.com/6269751/27990954-deb45a52-642c-11e7-8210-54a622f0b7f8.png)

**Comment by Waitsnake** [comment]
I also thought about a tiled mode long time ago when I first add different view modes.

But than you have to ask you first how many tiles you like to show?
2x2? 3x3? 4x3 ? ... 6x6? ... 9x9 ? Such an static solution is obviously easier to implement.
Or calculate just enough tiles around one in the middle of an start tile until the screen is filled? 
But how large you scale the first tile than? And in the end you have to calculate a bunch of coordinates for different scaled squares and than drop the textures multiple times on it.

When I saw the extra time, work and effort for this view option I dropped the idea of having it since it is only a nice to have from my point of view.

But the source code is free to change so you can participate the project if you like and make fork of it and than send me a pull-request if you implement an tile mode as you like it.

**Comment by MichaelDimmitt** [comment]
You can use applescript to get the size of the screen. That will make calculation easier.
@Waitsnake, I will look for my code when I did that. 
Hope you find this helpful:
```applescript
osascript -e 'tell application "Finder"
set screen_resolution to bounds of window of desktop
end tell'
```

Then from you calculations making the tiles with Objective C.

**Comment by Waitsnake** [comment]
I decide to implement an static 3x3 tiles mode that can be enabled/disabled by an check box button and it can be paired with all previous view options. It will be come with next commit.

**Comment by Waitsnake** [comment]
Feature is included in 1.4.0 (I made a silent upgrade of release 1.4.0 to the one 2 hours ago)!
I needed to restructure the hole drawing routine to implement this in an easy way.
I hope you like it. :-)
